### PR TITLE
feat: Add inventory page

### PR DESCRIPTION
**Pull request by irfankhaan**

### CODE REVIEW

**Comment by Haider8** [comment]
@irfankhaan Have you started integrating this with firebase?

**Comment by irfankhaan** [comment]
I tried doing this but it is showing me the following error.
Error: permission_denied at /inventory/microcontroller: Client doesn't have permission to access the desired data.

**Comment by iamareebjamal** [comment]
Fixing it

File: pages/inventory.vue [hunk 1]
```diff
@@ -47,38 +47,34 @@
...
-            'https://images-na.ssl-images-amazon.com/images/I/4133JwedpXL._SX466_.jpg'
+      title: 'Inventory'
+    }
+  },
+  computed: {
+    ...mapState('inventory', ['inventory'])
+  },
+  created() {
+    this.setInventoryRef({
+      ref: this.$firebase.database().ref('inventory/microcontroller'),
```
Comment (Haider8) [inline pages/inventory.vue]: @irfankhaan Fetch all inventory items and not just microcontroller.

**Comment by iamareebjamal** [comment]
Try in a few seconds. It should work

**Comment by irfankhaan** [comment]
I am still getting this error.
Error: permission_denied at /inventory: Client doesn't have permission to access the desired data.

**Comment by iamareebjamal** [comment]
Try now

File: pages/inventory.vue [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    <v-layout row wrap>
+      <v-card
+        v-for="inven in inventory.microcontroller"
+        :key="inven['.key']"
+        class="mx-auto mb-3"
+        width="400"
+      >
+        <v-flex xs12 class="text-xs-center">
+          <v-avatar :tile="false" :size="200" color="grey lighten-4 mt-3">
+            <v-img :aspect-ratio="16 / 9" :src="inven.thumbnail" alt="Avatar" />
```
Comment (Haider8) [inline pages/inventory.vue]: @irfankhaan There isn't any thumbnail key in the database.

Comment (irfankhaan) [reply pages/inventory.vue]: I was thinking I will add the thumbnail when I have to make a json for the actual inventory data. What do you say?

Comment (iamareebjamal) [reply pages/inventory.vue]: No need of thumbnail. Use DataTable component for the invrntory

**Comment by iamareebjamal** [comment]
Good work, but use single table, like this - https://deploy-preview-340--roboclub.netlify.com/downloads

**Comment by iamareebjamal** [comment]
Last thing, show the remaining items

File: pages/inventory.vue [hunk 3]
```diff
@@ -45,42 +45,48 @@
                   v-if="i == 1"
                   :headers="headers"
                   :items="inventory.microcontroller"
-                  hide-default-footer
                   class="elevation-1 mb-2"
                 >
                   <template v-slot:items="props">
                     <td>{{ props.item.name }}</td>
```
Comment (iamareebjamal) [inline pages/inventory.vue]: props is redundant. Remove `props.` from everywhere

Comment (irfankhaan) [reply pages/inventory.vue]: I tried removing the __props variable__ from v-slot but it's not working.

```
Property or method "item" is not defined on the instance but referenced during render. Make sure that this property is reactive, either in the data option, or for class-based components, by initializing the property.
TypeError: Cannot read property 'name' of undefined at inventory.js:339
```

**Pull request merged**